Sources/Tools/plPythonPack/PythonInterface.cpp

```diff
@@ -121,7 +122,25 @@ void PythonInterface::finiPython()
 //
 PyObject* PythonInterface::CompileString(const char *command, const plFileName& filename)
 {
-    PyObject* pycode = Py_CompileString(command, filename.AsString().c_str(), Py_file_input);
+    PyObject* filenameObj = PyUnicode_FromStringAndSize(
+        filename.AsString().c_str(),
+        filename.AsString().size()
+    );
+
+    PyCompilerFlags flags{};
+    flags.cf_feature_version = PY_MINOR_VERSION;
+
+    // Py_CompileString decodes the filename using the active codepage, but
```

Comment:
nit: correct Python terminology

```suggestion
    // Py_CompileString decodes the filename using the filesystem encoding, but
```